app/src/main/java/com/example/weightcontrolapp/ui/weightrecord/WeightRecordFragment.kt

```diff
@@ -0,0 +1,63 @@
+package com.example.weightcontrolapp.ui.weightrecord
+
+import android.os.Bundle
+import android.text.Editable
+import android.text.TextWatcher
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.EditText
+import androidx.fragment.app.activityViewModels
+import androidx.navigation.fragment.findNavController
+import com.example.weightcontrolapp.R
+import com.example.weightcontrolapp.R.layout.weight_record_fragment
+import com.google.android.material.snackbar.Snackbar
+
+class WeightRecordFragment : Fragment() {
+
+    private val viewModel: WeightRecordViewModel by activityViewModels()
+    private lateinit var weightRecordInput: EditText
+
+    private val textWatcher = object : TextWatcher {
+        override fun beforeTextChanged(s: CharSequence?, start: Int, count: Int, after: Int) {
+        }
+
+        override fun onTextChanged(s: CharSequence?, start: Int, before: Int, count: Int) {
+        }
```

Comment:
Are these codes necessary?

Reply:
These codes are necessary because these methods are the abstract method of TextWatcher interface.